microcloud/db/schema.go

```diff
@@ -0,0 +1,28 @@
+package db
```

Comment:
Why do we need a DB table for this?

The user won't be able to add new services, so if all we need is a map of service name to state directory, we can just bake that in microcloud directly as it will always be:
 - microceph => /var/snap/microceph/common/state/
 - lxd => /var/snap/lxd/common/lxd/

Reply:
So just make them all constants? In that case should we do that for the MicroCloud state directory on MicroCloud and MicroCeph directory on MicroCeph as well?

Reply:
They should be constants in microcloud. MicroCloud will always be run as a snap, technically the same is true for MicroCeph too but not for LXD.